spec/lib/mcb/editor/courses_editor_spec.rb

```diff
@@ -496,13 +604,13 @@ def run_new_course_wizard(*input_cmds)
           desired_attributes[:english],
           desired_attributes[:science],
           desired_attributes[:age_range],
+          desired_attributes[:level],
           desired_attributes[:course_code],
           'y', # is SEND confirmation
-          desired_attributes[:recruitment_cycle],
           "y", # confirm creation
           # subject selection
-          "Biology",
-          "Secondary",
+          "2",
+          "4",
```

Comment:
Do these numbers map to the subjects "Biology" and "Secondary"? Does that make this test order dependent? Is it worth doing that? Having the names of the subjects seemed nice and readable.